lib/modules/manager/gomod/artifacts.spec.ts

```diff
@@ -15,7 +15,17 @@ jest.mock('../../../util/exec/env');
 jest.mock('../../../util/git');
 jest.mock('../../../util/host-rules');
 jest.mock('../../../util/http');
-jest.mock('../../../util/fs');
+jest.mock('../../../util/fs', () => {
+  // restore
+  return {
+    __esModules: true,
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
+    ...(jest.createMockFromModule('../../../util/fs') as any),
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
+    isValidLocalPath: (jest.requireActual('../../../util/fs') as any)
```

Comment:
```suggestion
    ...(jest.createMockFromModule('../../../util/fs')),
    isValidLocalPath: jest.requireActual('../../../util/fs')
```
doesn't `requireActual` return any?

Reply:
yeah, that works on the assignment to `isValidLocalPath`, but  can't do the spread thing without the "as any" cast:

```
Spread types may only be created from object types.ts(2698)
```

Reply:
this makes the test fail actually:

```
jest.requireActual('../../../util/fs').isValidLocalPath,
```
with:
```
Object is of type 'unknown'.ts(2571)
```

---
this works:
```
 (jest.requireActual('../../../util/fs') as any).isValidLocalPath
```